packages/gravity/blade-runner/src/redis/redis.test.ts

```diff
@@ -5,23 +5,23 @@
 import { expect } from 'chai';
 import { Redis, type RedisOptions } from 'ioredis';
 
-import { type Message, cbor, type PeerId } from '@dxos/automerge/automerge-repo';
+import { type Message, type PeerId } from '@dxos/automerge/automerge-repo';
 import { type TaggedType } from '@dxos/codec-protobuf';
 import { EchoTestBuilder, TestReplicator, TestReplicatorConnection, createDataAssertion } from '@dxos/echo-db/testing';
 import { PublicKey } from '@dxos/keys';
-import { log } from '@dxos/log';
 import { type TYPES } from '@dxos/protocols';
 import { RpcPeer } from '@dxos/rpc';
 import { afterTest, describe, openAndClose, test } from '@dxos/test';
 
-import { REDIS_PORT, createRedisRpcPort } from './util';
+import { REDIS_PORT } from './defaults';
+import { createRedisReadableStream, createRedisRpcPort, createRedisWritableStream } from './util';
 
 /**
  * NOTE(mykola): This test is disabled because it requires a running Redis server.
  *               `redis-server --port 6378` to start a Redis server on port 6378.
  * TODO(mykola): Mock Redis server.
  */
-describe.skip('Redis', () => {
+describe('Redis', () => {
```

Comment:
forgot to skip, or do you start redis in the test?

Reply:
Just a mistake